[SSL] Geo Key Manager docs section (#7660)
cloudflare/cloudflare-docs: content/ssl/edge-certificates/geokey-manager/supported-options.md

---
pcx_content_type: reference
title: Supported options
weight: 2
layout: single
meta:
    title: Supported options - Geo Key Manager
---

# Supported options

## Available regions

For customers with Geo Key Manager v2, you can use the `policy` parameter to specify following regions using the **Region code**:

| Region code | Region name |
| --- | --- |
| AFR | Africa |
| EU | European Union |
| OC | Oceania |
| APAC | Asia Pacific |
| ME | Middle East |
| WEUR | Western Europe |
| SAM | South America |
| ENAM | Eastern North America |
| WNAM | Western North America |

---

## Available countries

> For customers with Geo Key Manager v2, you can use the `policy` parameter to individual countries as well. Cloudflare is constantly expanding the number of supported countries. To indicate a country, specify the two-letter (ISO 3166) country code.

Reviewer: ```suggestion
For customers with Geo Key Manager v2, you can use the `policy` parameter to specify individual countries as well. Cloudflare is constantly expanding the number of supported countries. To indicate a country, specify the two-letter (ISO 3166) country code.
```

Examples of supported countries are Japan, Canada, India, and Austrailia.

---

## Highest security data centers

For customers with both Geo Key Manager v1 and v2, you can use the `geo_restrictions` parameter to only choose Cloudflare's highest security data centers.

The following aspects are unique to our highest security data centers, but the baseline security requirements for all data centers are also detailed in [our blog](https://blog.cloudflare.com/introducing-cloudflare-geo-key-manager/).

> ### Pre-Scheduled, Biometric Controlled Facility Access

Reviewer: Not sure if we must keep the capitalization here or if these headings can use sentence case.

Employees of Cloudflare permitted to access the facility must have previously scheduled a visit before access will be granted.

Access to the entrance of the facility is controlled through the use of a biometric hand reader combined with an assigned access code.

### Private Cages with Biometric Readers

All equipment is in private cages with physical access controlled via biometrics and recorded in audit logs.

> Entrants have to pass through 5 separate readers before the cage can be accessed.

Reviewer: ```suggestion
Entrants have to pass through five separate readers before they can access the cage.
```
Hopefully, we can adjust this text...

### Exterior Security Controls and Monitoring

All points of ingress/egress are monitored by an intrusion detection system (IDS), with authorized users and access events archived for historical review.

### Interior Security Controls and Monitoring

Interior points of ingress/egress are controlled by the access control subsystem, with entry routed through a mantrap. All areas are monitored and recorded with closed-circuit television, with data kept for a minimum of thirty days.

> Exterior walls are airtight and may incorporate additional security measure such as reinforced concrete, Kevlar bullet board, vapor barriers, or bullet-proof front doors.

Reviewer: ```suggestion
Exterior walls are airtight and may incorporate additional security measures such as reinforced concrete, Kevlar bullet board, vapor barriers, or bullet-proof front doors.
```

---
pcx_content_type: reference
title: Supported options
weight: 2
layout: single
meta:
    title: Supported options - Geo Key Manager

> ---

Reviewer: We should probably have a description here also.
